src/full/Agda/Interaction/Highlighting/LaTeX.hs

```diff
@@ -485,29 +485,29 @@ processCode toks' = do
       Name (Just kind) isOp ->
         (\c -> if isOp then ["Operator", c] else [c]) $
         case kind of
-          Bound                     -> s
-          Generalizable             -> s
+          Bound                     -> sk
+          Generalizable             -> sk
           Constructor Inductive     -> "InductiveConstructor"
           Constructor CoInductive   -> "CoinductiveConstructor"
-          Datatype                  -> s
-          Field                     -> s
-          Function                  -> s
-          Module                    -> s
-          Postulate                 -> s
-          Primitive                 -> s
-          Record                    -> s
-          Argument                  -> s
-          Macro                     -> s
+          Datatype                  -> sk
+          Field                     -> sk
+          Function                  -> sk
+          Module                    -> sk
+          Postulate                 -> sk
+          Primitive                 -> sk
+          Record                    -> sk
+          Argument                  -> sk
+          Macro                     -> sk
         where
-        s = show kind
+        sk = show kind
 
 -- | Escapes special characters.
 escape :: Text -> Text
 escape (T.uncons -> Nothing)     = T.empty
 escape (T.uncons -> Just (c, s)) = T.pack (replace c) <+> escape s
   where
   replace :: Char -> String
-  replace c = case c of
+  replace char = case char of
```

Comment:
Better: use `\case`.

Reply:
Ah, yep. I agree `\case` would be better here, will fix.

Also, I am 100% behind not squashing these. The commits are broken up to maximize automatic mergeability if there are any overlapping unpublished changes out there.

Reply:
Oh! Actually, this can't be `\case`: the argument is used in the fallback pattern below: https://github.com/agda/agda/pull/5016/files/e5c06660b6e34dbce605a03be639646e1aaec750#diff-8baff3a78102c97460fea4072aa1fb914b189432cceff772b11463737fba83ebR521